src/app/users/views/editServices.ejs

```diff
@@ -32,7 +32,7 @@
 
                         <% if (locals.serviceRoles.length > 0) { %>
                             <legend class="govuk-fieldset__legend govuk-fieldset__legend--xs">
-                                <p class="govuk-body">Some services ask you to select a sub-service, role, or form to better define how you will use them.
+                                <p class="govuk-body">Select 1 role. Any changes you make to sub-service selection will change access for <%= locals.user.firstName %> <%= locals.user.lastName %>
```

Comment:
Can we add a full stop at the end of the sentence?

Reply:
`Select 1 role.` - this doesn't sound right - there are services which allow selecting multiple roles - could you please double check with the content designer?

Reply:
@ipredoi Yes it's doesn't seem quite right but just following the AC. But I can double check with Heather.